.github/workflows/publish_latest_graphrag_image.yml

```
@@ -0,0 +1,85 @@
+    env:
+      REPOSITORY_URL: apache/incubator-hugegraph-ai
+      BRANCH: master
+      IMAGE_URL: hugegraph/graphrag:latest
+      IMAGE_URL_NK: hugegraph/graphrag_nk:latest
+      GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+      OWNER: hugegraph
+      REPO: actions
+      LAST_AI_HASH: ${{vars.LAST_AI_HASH}}
+      
+    steps:
+    - name: Set up Docker Buildx
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)